frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -588,6 +595,76 @@ Except when in two columns mode, where this is limited to showing only the previ
             self.allow_corner_scroll = G_reader_settings:nilOrTrue("highlight_corner_scroll")
         end,
     })
+    -- we allow user to select the rate at which the content selection tool moves through screen
+    if not Device:isTouchDevice() and Device:hasDPad() then
+        table.insert(menu_items.long_press.sub_item_table, {
+            text_func = function()
+                return T(_("Rate of movement in content selection: %1"), G_reader_settings:readSetting("highlight_non_touch_factor", 4))
+            end,
+            callback = function(touchmenu_instance)
+                local SpinWidget = require("ui/widget/spinwidget")
+                local curr_val = G_reader_settings:readSetting("highlight_non_touch_factor", 4)
+                local spin_widget = SpinWidget:new{
+                    value = curr_val,
+                    value_min = 0.25,
+                    value_max = 5,
+                    precision = "%.2f",
+                    value_step = 0.25,
+                    default_value = 4,
+                    title_text = _("Rate of movement"),
+                    info_text = _("Select a decimal value from 0.25 to 5, smaller values result in greater travel per keystroke. Font size and this value are inversely correlated, i.e smaller font size requires a larger value and vice versa."),
```

Comment:
This is new text? Maybe something a bit more like this.

```suggestion
                    info_text = _("Select a decimal value from 0.25 to 5. A smaller value results in a larger travel distance per keystroke. Font size and this value are inversely correlated, meaning a smaller font size requires a larger value and vice versa."),
```